.github/workflows/build_release_candidates.yml

```diff
@@ -37,3 +37,58 @@ jobs:
           name: client_linux_release_${{ github.sha }}
           path: build/dist
           if-no-files-found: error
+  android:
+    name: Build Android Release Candidate
+    runs-on: ubuntu-20.04
+    timeout-minutes: 15
+    environment: Client Android Release
+    env:
+      SENTRY_DSN: ${{ secrets.SENTRY_DSN }}
+      ANDROID_KEY_STORE_PASSWORD: ${{ secrets.ANDROID_KEY_STORE_PASSWORD }}
+      ANDROID_KEY_STORE_CONTENTS: ${{ secrets.ANDROID_KEY_STORE_CONTENTS }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+
+      - name: Install Node
+        uses: actions/setup-node@v2.2.0
+        with:
+          node-version: 16
+
+      - name: Install NPM Dependencies
+        run: npm ci
+
+      - name: Install Java
+        uses: actions/setup-java@v1.4.3
+        with:
+          java-version: 1.8
+
+      - name: Setup Android
+        run: bash ./tools/build/setup_linux_android.sh
+
+      - name: Build Android Release Candidate
+        run: |
+          cat "$ANDROID_KEY_STORE_CONTENTS" > keystore.p12
```

Comment:
`cat` takes a filename. How is this working?
I'm pretty sure you need `echo` here instead.